gui/js/updater.window.js

```diff
@@ -116,23 +123,23 @@ module.exports = class UpdaterWM extends WindowManager {
   }
 
   checkForUpdates() {
-    log.info('Looking for updates...')
     this.timeout = setTimeout(() => {
```

Comment:
What happens when creating new timeouts on each call?

Reply:
It would be a pain, but I don't think the current PR changes anything regarding this.

Reply:
Nevermind. I thought we were calling this method multiple times.